File: FitToFit/FitToFit.Model/Treneri.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FitToFit.Model
{
    public class Treneri
    {
        public int TrenerId { get; set; }

        public string Ime { get; set; } = null!;

        public string Prezime { get; set; } = null!;

        public string Spol { get; set; } = null!;

        public string Telefon { get; set; } = null!;

        public string Email { get; set; } = null!;

        public string Adresa { get; set; } = null!;

        public DateTime DatumZaposlenja { get; set; }

        public string Zvanje { get; set; } = null!;

        public byte[]? Slika { get; set; }
    }
}

File: FitToFit/UI/fittofit_admin/lib/models/treneri.dart
import 'package:json_annotation/json_annotation.dart';

import 'ocjene.dart';

part 'treneri.g.dart';

@JsonSerializable()
class Treneri {
  int trenerId;
  String ime;
  String prezime;
  String spol;
  String? telefon;
  String? email;
  String? adresa;
  DateTime datumZaposlenja;
  String? zvanje;
  String? slika;
  List<Ocjene>? ocjene;

  Treneri(
      {required this.trenerId,
      required this.ime,
      required this.prezime,
      required this.spol,
      this.telefon,
      this.email,
      this.adresa,
      required this.datumZaposlenja,
      this.zvanje,
      this.slika,
      this.ocjene});

  factory Treneri.fromJson(Map<String, dynamic> json) =>
      _$TreneriFromJson(json);

  Map<String, dynamic> toJson() => _$TreneriToJson(this);
}
